alpine/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG ARCH=""
+FROM        ${ARCH}alpine:latest as alpine
+
+RUN apk upgrade -U && \
+    apk add tzdata ca-certificates busybox-static
+
+# remove extra package
+RUN apk del libc-utils
+
+# remove apk files and directories
+RUN apk del apk-tools && \
+    find / -name apk -prune -exec rm -rf {} ";"
+
+# Use the busybox.static to avoid dynamic library dependencies.
+RUN mv /bin/busybox.static /bin/busybox && \
+    /bin/busybox --install -s /bin
+
```

Comment:
Previous iterations were trying to not include musl-libc .  In this iteration, I haven't remove it.  If that is okay, then we can leave the default dynamic linked `busybox`.  

```suggestion
```